app/controllers/books_controller.rb

```diff
@@ -6,4 +6,12 @@ def index
   def show
     @book = Book.find(params[:id])
   end
+
+  def destroy
+    @book = Book.find(params[:id])
+    @book.reviews.destroy_all
+    @book.destroy
```

Comment:
if we specify `dependent: :destroy`, we don't have to delete `reviews` by ourself. Rails will do it for us.
see comment on: https://github.com/giftnichakul/practice_ruby_on_rails/pull/6
```suggestion
    @book.destroy
```